### ISSUE TITLE
train custom dataset problem

### ISSUE THREAD

**Issue by bloom1123** [google/automl#1125]
I meet the flowing problem, i don't know how to solve it.
2 root error(s) found.
  (0) INVALID_ARGUMENT: Input size should match (header_size + row_size * abs_height) but they differ by 2
	 [[{{node parser/decode_image/DecodeImage}}]]
	 [[IteratorGetNext]]
	 [[IteratorGetNext/_14387]]
  (1) INVALID_ARGUMENT: Input size should match (header_size + row_size * abs_height) but they differ by 2
	 [[{{node parser/decode_image/DecodeImage}}]]
	 [[IteratorGetNext]]
0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

  File "/home/user/Downloads/automl-master/efficientdet/main.py", line 344, in run_train_and_eval
    max_steps=e * FLAGS.num_examples_per_epoch // FLAGS.train_batch_size)
  File "/home/user/Downloads/automl-master/efficientdet/main.py", line 360, in main
    run_train_and_eval(e)
  File "/home/user/Downloads/automl-master/efficientdet/main.py", line 367, in <module>
    app.run(main)

**Comment by fsx950223** [comment]
Something wrong with your images.

**Comment by mondrasovic** [comment]
Yes, it is true that there is something wrong with the dataset. I do not know which dataset was used by you, nevertheless, I identified the root cause and documented it in my [StackOverflow response](https://stackoverflow.com/questions/69607117/tensorflow-invalid-argument-input-size-should-match-but-they-differ-by-2/73004159#73004159).
